index.js

```diff
@@ -1,3 +1,6 @@
+// workaround for https://github.com/nodejs/node/issues/16196
+require('tls').DEFAULT_ECDH_CURVE = 'auto'
```

Comment:
I'm guessing you tested this and it fixed the issue.  I'm a little confused why though since it looks like "auto" is the default value [according to the docs](https://nodejs.org/api/tls.html#tls_tls_default_ecdh_curve).

Reply:
Thanks for the explanation!